hw/top_earlgrey/data/pins_cw341.xdc

```diff
@@ -0,0 +1,135 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## ChipWhisperer CW341 "Bergen Board".
+##
+
+## Clock Signal
+set_property -dict { PACKAGE_PIN E22 IOSTANDARD LVCMOS18 } [get_ports { IO_CLK }]; # PLL_CLK2
+
+## Clock constraints
+## set via clocks.xdc
+
+## Power-on Reset
+set_property -dict { PACKAGE_PIN  G26 IOSTANDARD LVCMOS18 PULLTYPE PULLUP } [get_ports { POR_N }]; # Main PORN, requires jumper to SW0 [was USB_A14 (SAM3X)]
+
+## Preserve prim_prince modules and setup multi-cycle paths
+## These are no longer required, but kept here as a reference
+## set_property KEEP_HIERARCHY TRUE [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]
+## set_multicycle_path -setup 2 -through [get_pins -of_objects [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]]
+## set_multicycle_path -hold 1  -through [get_pins -of_objects [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]]
+
+#set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets IO_SDCK_IBUF]; # SDCK clock to be ignored
+
+
```

Comment:
I would suggest following the CW310 [comment structure](https://github.com/lowRISC/opentitan/blob/189dab9f7dab6f74c4c42b88b0eb49ebfeeddfcd/hw/top_earlgrey/data/pins_cw310.xdc#L38)

Reply:
I added comments to reflect how each pin is intended to be used. However, CW340 has a bunch of headers and different options, and I balked at reproducing netlists in this file.